static/templates/stream_member_list_entry.hbs

```diff
@@ -3,7 +3,7 @@
     {{#if show_email}}
```

Comment:
We'll want to put some indication of which users are stream administrators in this table.